resources/definitions/creality cr-x.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "CR-X",
+    "version": 2,
+    "name": "Creality3D CR-X",
```

Comment:
Please use `Creality CR-X` instead to keep consistency with the other printers.
```suggestion
    "name": "Creality CR-X",
```

<img width="155" alt="screenshot 2019-02-01 at 18 04 55" src="https://user-images.githubusercontent.com/15830521/52137888-f832eb00-264b-11e9-8495-cc5adef58707.png">

Reply:
Updated